packages/waveform-progress/package.json

```diff
@@ -5,6 +5,8 @@
   "license": "AGPL-3.0-only",
   "main": "index.js",
   "module": "index.js",
+  "types": "index.d.ts",
+  "preserveSymlinks": "true",
```

Comment:
what is this toggle for? lerna?

Reply:
Oh I forgot to remove this, but yeah since you use symlinks in Lerna, there's a flag you have to add when you're using symlinks in your node_modules directory to tell the server to treat the symlinks as regular directories. Otherwise the server gives you an error that the files are outside of the current directory. I don't actually need this so I removed it.